zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -99,17 +132,7 @@ class MeasurementType(enum.IntFlag):
         ),
     )
     ZCL_INIT_ATTRS = {
-        ElectricalMeasurement.AttributeDefs.ac_current_divisor.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_current_multiplier.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_power_divisor.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_power_multiplier.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_voltage_divisor.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_voltage_multiplier.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_frequency_divisor.name: True,
-        ElectricalMeasurement.AttributeDefs.ac_frequency_multiplier.name: True,
         ElectricalMeasurement.AttributeDefs.measurement_type.name: True,
-        ElectricalMeasurement.AttributeDefs.power_divisor.name: True,
-        ElectricalMeasurement.AttributeDefs.power_multiplier.name: True,
         ElectricalMeasurement.AttributeDefs.power_factor.name: True,
```

Comment:
not related to this PR, but why do we cache power_factor?

Reply:
Good question! Looking at ZCL spec, `power_factor` isn't reportable though.. So I guess that has to be polled then.